README.md

```diff
@@ -40,8 +40,8 @@ The Mongo ODM for Go
 - `mgm` wraps the official Mongo Go Driver.
 
 ## Requirements
-- Go 1.10 or higher.
-- MongoDB 2.6 and higher.
+- Go 1.17 or higher.
+- MongoDB 4.2 and higher.
```

Comment:
Thanks for your contribution @trungdlp-wolffun :))
You've updated the monog-driver's version to `v1.10.3` in `go.mod` file, that version of Mongo official driver [supports](https://github.com/mongodb/mongo-go-driver/tree/v1.10.3) MongoDB 2.6 or higher, so actually we support MongoDB 2.6 or higher, unless you update the mongo driver's version to its latest version and then change this line to [MongoDB 3.6 and higher](https://github.com/mongodb/mongo-go-driver/tree/master#requirements)